include/crypto/x509.h

```diff
@@ -318,3 +318,5 @@ int X509_add_cert_new(STACK_OF(X509) **sk, X509 *cert, int flags);
 
 int X509_PUBKEY_get0_libctx(OSSL_LIB_CTX **plibctx, const char **ppropq,
                             const X509_PUBKEY *key);
+/* Calculate default key identifier according to RFC 5280 section 4.2.1.2 (1) */
+ASN1_OCTET_STRING *X509_PUBKEY_hash(X509_PUBKEY *pubkey);
```

Comment:
Please use x509_pubkey_hash as name as this is an internal function.

Reply:
Done